src/plugins/intel_gpu/tests/fusions/deconvolution_fusion_test.cpp

```diff
@@ -629,6 +629,42 @@ INSTANTIATE_TEST_SUITE_P(fusings_gpu, deconv_scale_actv_quant_i8, ::testing::Val
     deconv_test_params{ CASE_DECONV_S8S8_3D_8, 2, 5 },
 }));
 
+TEST(fusings_gpu, deconv_scale_actv_softsign) {
+    auto& engine = get_test_engine();
+
+    auto input = engine.allocate_memory({data_types::f32, format::yxfb, {1, 1, 2, 2}});
+    auto weights = engine.allocate_memory({data_types::f32, format::oiyx, {1, 1, 2, 2}});
+    auto biases = engine.allocate_memory({data_types::f32, format::bfyx, {1, 1, 1, 1}});
+    auto scale_data = engine.allocate_memory({data_types::f32, format::bfyx, {1, 1, 1, 1}});
+
+    set_values(input, {8.f, 0.5f, 6.f, 9.f});
+    set_values(weights, {-2.0f, 0.5f, 3.5f, 1.5f});
+    set_values(biases, {2.0f});
+    set_values(scale_data, {2.0f});
+
+    topology topo(input_layout("input", input->get_layout()),
+                      data("weights", weights),
+                      data("biases", biases),
+                      data("scale_data", scale_data),
+                      deconvolution("deconv", "input", {"weights"}, {"biases"}, {1, 1}),
+                      eltwise("scale", {"deconv", "scale_data"}, eltwise_mode::prod),
+                      activation("actv", "scale", activation_func::softsign),
```

Comment:
I think in this case, we don't need to make a separate test case for `softsign`. Could you check why you've changed softsign to hyptan?

Reply:
I've changed activation function to hypbtan because softsign does not supported in onednn activation. Current PR is using softsign with adjustment of p.expected_fused_primitives value instead of changing activation function.